### DISCUSSION TITLE
parking_lot v0.12 vs MSRV

### DISCUSSION THREAD

**Discussion by olix0r** [tokio-rs/tokio#4456] [General] [answered]
There's a new version of `parking_lot`, v0.12, that bumps its MSRV to 1.49 (which was released >1 year go). As ecosystem projects update to use this new version of `parking_lot`, this will create duplicate dependencies for many projects.

Are there objections to updating Tokio to use the new version of parking_lot and bumping MSRV to 1.49? I'm happy help to make this change if it will be accepted.

@carllerche @Darksonn

**Comment by carllerche** [comment, marked as answer]
It should be fine to bump hte MSRV to 1.49. I would start w/ a PR doing just that.
